modules/recon/hosts-hosts/ipinfodb.py

```diff
@@ -11,10 +11,15 @@ class Module(BaseModule):
         'description': 'Leverages the ipinfodb.com API to geolocate a host by IP address. Updates the \'hosts\' table with the results.',
         'required_keys': ['ipinfodb_api'],
         'query': 'SELECT DISTINCT ip_address FROM hosts WHERE ip_address IS NOT NULL',
+        'options': (
```

Comment:
Rather than all of this extra logic, if there is a static rate limit, why even have an option? Just delay 0.5 seconds for every request.

Reply:
I figured we could accommodate people that did pay for the service there is not a limit for those people. Either way, I don't think the limits make that much of an issue so pull incoming.